src/a11y.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {setStyle} from './style';
+
+/**
+ * Creates an off-screen button that is invisible visually but accessible via
+ * screen-readers.
+ * @param {!Element} parent to add the button to. Normally the AMP element.
+ * @param {string} label text for the button/
+ * @param {function()} onClick click handler.
+ * @return {!Element} Reference to the added button element.
+ */
+export function addScreenReaderButton(parent, label, onClick) {
+  const doc = parent.ownerDocument;
+  const screenReaderButton = doc.createElement('button');
+  screenReaderButton.textContent = label;
+  screenReaderButton.classList.add('-amp-screen-reader');
+  // This is for screen-readers only, should not get a tab stop.
+  screenReaderButton.tabIndex = -1;
```

Comment:
@marcysutton Both TalkBack and VoiceOver seem use the DOM order in single-finger swipes to navigate between items on the page and tab order does not seem to impact their discovery of these "invisible" buttons in both "default" and "controls" modes. 

Please note that we have alternatives for desktop (for example Carousel never hides the Next/Previous buttons on non-touch devices and those buttons stays in proper tab order for keyboard-only users).

On a side note, thanks for advocating for A11Y and we would love to collaborate with industry experts like yourself on A11Y in AMP. Are you using AMP for any of your project? If you encounter any A11Y issues in AMP, please let me know so I can make sure we prioritize pressing issues.